DEVELOPERS.md

```diff
@@ -18,9 +18,9 @@ To enable various gameplay tweaks, modify `game_config.json` in the root `cronen
 
 - `num_rooms`: an integer specifying how many rooms to place in the floor.
 - `extra_commands`: a Boolean specifying whether to add various extra commands, including [cheat](#cheats).
-- `random_run`: a Boolean specifying whether to let the game play itself by producing random commands.
+- `random_run`: an integer specifying the number of rounds the game should randomly play itself, or None.
 
-`num_rooms` can be helpful to restrict the size of randomly generated floors when testing. `extra_commands`'s primary utility lies in the [cheat](#cheats) command, which allows the player to alter the player character's statistics and abilities. `random_run` is mostly for fun.
+`num_rooms` can be helpful to restrict the size of randomly generated floors when testing. `extra_commands`'s primary utility lies in the [cheat](#cheats) command, which allows the player to alter the player character's statistics and abilities. `random_run` is mostly for fun and fuzzing.
```

Comment:
I didn't mean "mostly for fun" as damnation with faint praise, btw.

I did it for the rhymesss

Reply:
Oh shit, and now I've screwed it up!

Reply:
Or you nailed it. I think it's jazzier now.

:saxophone: